app/models/forms/bulk_upload_lettings/prepare_your_file.rb

```diff
@@ -9,7 +9,12 @@ class PrepareYourFile
       attribute :needstype, :integer
 
       def view_path
-        "bulk_upload_lettings_logs/forms/prepare_your_file"
+        case year
+        when 2022
+          "bulk_upload_lettings_logs/forms/prepare_your_file_2022"
+        when 2023
```

Comment:
Is it worth having these fallback gracefully when year is not 2022 or 2023?

Reply:
i can make it catch all to 2023. it should be noted unfortunately at this stage the form is state dependent. if year is missing the back button won't work. but your change would at least make new years compatible without a future change